src/main/scala/io/epiphanous/flinkrunner/model/sink/JdbcSinkConfig.scala

```diff
@@ -177,6 +177,25 @@ case class JdbcSinkConfig[ADT <: FlinkEvent](
     case _             => Seq.empty
   }
 
+  val isTimescale: Boolean = config
+    .getObjectOption(pfx("table.timescale"))
+    .nonEmpty
+
+  val timescaleTimeColumn: Option[String]   = config
+    .getStringOpt(pfx("table.timescale.time.column"))
+
+  val timescaleChunkTimeInterval: String = config
+    .getStringOpt(pfx("table.timescale.chunk.time.interval"))
+    .getOrElse(DEFAULT_TIMESCALE_CHUNK_TIME_INTERVAL)
+
+  val timescalePartitioningColumn: String = config
```

Comment:
I would just keep this as an `Option[String]`, and then process it appropriately below at L499.

Reply:
I agree, incorporated the change.